### PR TITLE
Syndicate Revolver/357 Ammo Nerfs

### PR DESCRIPTION
**Pull request by wesoda25**
## About The Pull Request

Gives Syndicate Revolver a minor recoil (screen shake), and 357 ammo is worse at armored opponents. 

## Why It's Good For The Game

Syndicate Revolver is a cancer weapon that somehow has made it this far without further nerfs, probably due to the honor (or ignorance) of players to not buy it. It doesn't matter whether or not you have HOS armor or are unarmed, you'll be dropped in 3 shots tops, and unlimited ammo can be printed. Its by far the strongest weapon you can purchase in the uplink, and is really unfair considering the current disabler nerfs. My PR makes it a bit harder to aim, and gives security members a fighting chance by making the bullets a bit worse against armor. 

## Changelog
:cl:
balance: The Syndicate Revolver now has a recoil
balance: 357 ammo is now worse at penetrating armor
/:cl:

### CODE REVIEW

**Comment by imsxz** [comment]
ded

**Comment by kittymaster0** [comment]
Nick Johnson killed me please nerf.

**Comment by wesoda25** [comment]
I didn't even play the round you're thinking of, just heard of it. No salt here bb, just balances

**Comment by imsxz** [comment]
i havent played today idk what round you think im thinking of

**Comment by imsxz** [comment]
this gives all revolver subtypes including det gun a recoil

**Comment by wesoda25** [comment]
oo one sec

**Comment by zxaber** [comment]
>Gives Syndicate Revolver a minor recoil (screen shake)

.44 Mag revolver when

**Comment by habader** [comment]
ided

**Comment by AdamElTablawy** [comment]
could also give the revolver armor pen ammo versions as well, for armored targets, like the stetchkin does.

minor screenshake seems fine, though.

**Comment by Eaglendia** [comment]
Have you tried not getting shot?

**Comment by AdamElTablawy** [comment]
have you tried not getting tazed?

oh, wait

**Comment by wesoda25** [comment]
> Have you tried not getting shot?

I actually never thought of it this way, thanks for the second opinion bro closing PR now

**Comment by wesoda25** [comment]
ok i fixed it

File: code/modules/uplink/uplink_items.dm [hunk 1]
```diff
@@ -456,7 +456,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	cost = 2
 	include_modes = list(/datum/game_mode/nuclear)
 
-/datum/uplink_item/dangerous/revolver
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/uplink/uplink_items.dm]: you changed the part that doesn't need to be changed hehe

Comment (wesoda25) [reply code/modules/uplink/uplink_items.dm]: wdym? It'd just revert to normal revolver in that state, with no recoil.

**Comment by Eaglendia** [comment]
Admittedly, they can win out over stun batons as close-ranged ambush weapons (I'm pretty sure 42 damage applies slowdown) - but their telecrystal cost, even for a weapon that has good lethality in addition to this, is pretty prohibitive without discounts. There's a lot of factors that'd go into determining objectively where they stand competitively against disablers in skirmish encounters.

I don't think .357 ammunition necessarily needs a special negative armor penetration factor when bulletproof armor can be distributed at any point. It takes five shots to crit through this and exists for the purpose of being handed out to cockblock ballistics murderboners.

**Comment by wesoda25** [comment]
You realize that all you need is a revolver (plus maybe flashbang protection) to murderbone so long as you have some adequate amount of skill? I'd argue its a very, very low cost to pay for the insane power you get, considering you can print infinite ammo. 

My PR basically gives people with varying armor +1 shots that they can tank before dropping, giving sec members not wearing the armory locked bulletproof armor a fighting chance.

**Comment by Tlaltecuhtli** [comment]
dont listen to sybil admins they are stuck to farmville rounds

File: code/modules/uplink/uplink_items.dm (hunk 1 above)
Comment (nemvar) [reply code/modules/uplink/uplink_items.dm]: You need to change line 462 if you want to change the item you get, not this one.

Comment (ExcessiveUseOfCobblestone) [reply code/modules/uplink/uplink_items.dm]: this is the datum for the entry in the uplink, it doesn't change the item dispensed (aka the `item` var of this)

**Comment by FloranOtten** [comment]
Also, a full mag/revolution/speedloader/packet of bullets from this causes the SM to go into stabilization. It isn't a large delam but it can still destroy engine room without any chance to save it. Can we get that nerfed?

File: code/modules/uplink/uplink_items.dm (hunk 1 above)
Comment (wesoda25) [reply code/modules/uplink/uplink_items.dm]: yeah no im a dummy when i looked at the code the first time i knew to put it there but when i changed this i wasnt paying attention

**Comment by GhostAvenger** [comment]
Ided please nerf

**Comment by wesoda25** [comment]
Literally haven't been killed by a syndi revolver for something like 6 months I'm pretty sure. Y'all got me, ever since that time I've been waiting, simmering, until I could finally take out my hatred and anger on the player who killed me.

**Comment by 4dplanner** [comment]
have you considered




















I ded?

**Comment by necromanceranne** [comment]
Man, revolvers were fun to take as nukie just because they're kind of arse compared to their other gear. I don't suppose you could add an AP ammo to the nukie uplink could you? Compared to their other guns it's hardly much stronger since the TC cost is really ridiculous for them.

**Comment by ATH1909** [comment]
"you'll be dropped in 3 shots tops"
Doesn't a disabler also drop you in 3-4 shots?

**Comment by necromanceranne** [comment]
They're broken at the moment so not even :3

**Comment by Garen7** [comment]
.357 is currently nuts, if you don't think it is then you should really just play against it, I don't know the round wesoda's talking about but I have plenty experience in other rounds. It was fine when tasers one shot you, but now disablers 3 shot you so now you have to be some kind of legend to win that fight(assuming the .357 starts the fight)

**Comment by Dawson1917** [comment]
By all means nerf the damage of the 13tc handgun but for the love of god don't add screen shake. it's obnoxious, not pleasant to look at, and isn't present on the majority of other weapons including riot and combat shotguns

**Comment by wesoda25** [comment]
> By all means nerf the damage of the 13tc handgun but for the love of god don't add screen shake. it's obnoxious, not pleasant to look at, and isn't present on the majority of other weapons including riot and combat shotguns

no

**Comment by wesoda25** [comment]
Like I get what you’re saying, the screen shake could definitely get annoying at times. But I think selectively damage nerfs aren’t enough, let alone are boring as shit. And while I don’t care about murderboning, I know a lot of people want code solutions to lessen the action. Simply put, if you don’t want your screen to shake a lot don’t use the gun a lot. 

But honestly its not that bad.

**Comment by wesoda25** [comment]
Also @AdamElTablawy I don’t wanna add different ammo types, especially not armor piercing ones. I think a flat ammo type is good, its certainly easier to balance around.

**Comment by daklj** [comment]
> nerf stuns becasue stun weapons > pure damage weapons
> pure damage weapons get viable
> nerf pure damage weapons because now pure damage > stun
literally you

**Comment by PKPenguin321** [comment]
consider that revolvers are strong on purpose

**Comment by Tlaltecuhtli** [comment]
m1911 ammo was changed for the exactly same issue (2/3 clicks to down people) and it improved the game

**Comment by AdamElTablawy** [comment]
Well, I guess stetchkins will finally have a purpose now. Maybe.

**Comment by ExcessiveUseOfCobblestone** [comment]
stop posting wrongthink

also getting motion sickness IRL from firing a gun in spacesim is the ultimate balance

**Comment by wesoda25** [comment]
> consider that revolvers are strong on purpose

They’re still very strong, it really is a minor balance change.

**Comment by PKPenguin321** [comment]
You've framed your reasoning in the OP as though revolvers being strong is an oversight

**Comment by wesoda25** [comment]
No I framed it as if revolvers are fucking bullshit and require little to no skill to go on a murder spree with. My primary criticism of them is that sec basically has very little chance to win, especially if the traitor starts shooting first (which they always do, basically why we removed tasers). My PR will give security a fighting chance, and make it slightly harder to aim the gun if you are shooting constantly. Revolver is still strong as shit, its just not broken anymore. So yes, I think them being as strong as they currently are is an oversight, and this PR will put them at a level which was intended (still very strong!).

**Comment by PKPenguin321** [comment]
>So yes, I think them being as strong as they currently are is an oversight

Wrong

**Comment by wesoda25** [comment]
PKP im sorry that im making it so that you can't kill half the station using 13 TC and an autolathe

**Comment by ATH1909** [comment]
Honestly, I'd be fine with this PR if it was just the screenshake and not the armor piercing penalty.

**Comment by necromanceranne** [comment]
Just shoot people in the head. The only thing it can't kill as quickly is sec and a fully equipped validhunter, which probably isn't it's purpose and we have the stechkin with AP ammo for that.

**Comment by sagaty13** [comment]
salty coder wants to remove fun because it killed him
typical

**Comment by Steelpoint** [comment]
The reason this has never really been discussed is because it touches on armour. If you remember your history, traitor guns and armour values have been a hotly controversial topic for a long, long, time. 

Now, in my opinion. I think a better way to balance this is to remove the ability to print infinite Revolver ammo from the autolathe. 

The revolver remaining a very powerful weapon, that takes a lot of TCs to buy and load with ammo, seems more justifiable. Your sacrificing most of your TCs for a powerful, and very loud, ammo with a limited amount of bullets. 

I also think, but cannot confirm, that most sec armour gives you sufficient defence that you need 1 extra bullet to go down. 1 bullet when ammo is limited, is a big ask.

**Comment by wesoda25** [comment]
> I also think, but cannot confirm, that most sec armour gives you sufficient defence that you need 1 extra bullet to go down. 1 bullet when ammo is limited, is a big ask.

I tested and didn’t see anything myself that would support this.

**Comment by Steelpoint** [comment]
I figured since those armour pen values were always a bit hard to pin down.

**Comment by wesoda25** [comment]
Actually I want to do some testing, I think I could do a better nerf.

**Comment by wesoda25** [comment]
Actually I'm baffled right now, messing around with armor pen values and I find theres 0 difference in damage done? Really confused on this one this will take a while.

**Comment by sagaty13** [comment]
This isn't getting approved, fuck off man
The community doesn't want it

**Comment by AdamElTablawy** [comment]
what the playerbase wants is pretty much unimportant, since 90% of players are retards

**Comment by wesoda25** [comment]
lol his active repo is hippie, who woulda thunk it

**Pull request closed without merging**